pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/__specs__/VenueEdition.spec.jsx

```diff
@@ -392,6 +393,42 @@ describe('test page : VenueEdition', () => {
       })
     })
 
+    it('should be able to edit when venue does not have reimbursement point', async () => {
+      const storeOverrides = {
+        features: {
+          list: [
+            {
+              nameKey: 'ENABLE_NEW_BANK_INFORMATIONS_CREATION',
+              isActive: true,
+            },
+          ],
+        },
+      }
+      jest.spyOn(api, 'getAvailableReimbursementPoints').mockResolvedValue([
+        {
+          iban: 'FR0000000000000001',
+          name: 'Business unit #1',
+          venueName: 'Reimbursement point #1',
+        },
+      ])
+
+      await renderVenueEdition({ props, storeOverrides })
+
+      fireEvent.change(await screen.findByLabelText('Nom d’usage du lieu :'), {
+        target: { value: 'Mon lieu' },
+      })
```

Comment:
On essaie de s'affranchir de l'utilisation de fireEvent au profit de userEvent dans les tests, on pourrait ici fait quelque chose comme ça :

```suggestion
     const publicNameInput = await screen.findByLabelText(
        'Nom d’usage du lieu :'
      )
      await userEvent.clear(publicNameInput)
      await userEvent.type(publicNameInput, 'Mon lieu')
```